app/assets/javascripts/darkswarm/controllers/checkout/checkout_controller.js.coffee

```diff
@@ -9,9 +9,18 @@ Darkswarm.controller "CheckoutCtrl", ($scope, storage, Checkout, CurrentUser, Cu
   for field in $scope.fieldsToBind
     storage.bind $scope, "Checkout.order.#{field}",
       storeName: "#{prefix}_#{field}"
+
   storage.bind $scope, "Checkout.ship_address_same_as_billing",
     storeName: "#{prefix}_sameasbilling"
-    defaultValue: true
+    defaultValue: 'YES'
```

Comment:
What is the reason for switching from true/false to 'YES'/'NO'? I think it's fine, but there must have been a reason for doing it?